Library/Source/NativeXr.cpp

```diff
@@ -8,16 +8,16 @@
 
 namespace
 {
-    bgfx::TextureFormat::Enum XrTextureFormatToBgfxFormat(xr::TextureFormat format)
+    std::pair<bgfx::TextureFormat::Enum, uint64_t> XrTextureFormatToBgfxFormat(xr::TextureFormat format)
     {
         switch (format)
         {
         case xr::TextureFormat::RGBA8:
-            return bgfx::TextureFormat::RGBA8;
-        case xr::TextureFormat::RGBA8S:
-            return bgfx::TextureFormat::RGBA8S;
+            return{ bgfx::TextureFormat::RGBA8, BGFX_TEXTURE_RT };
+        case xr::TextureFormat::RGBA8_NONLINEAR:
+            return{ bgfx::TextureFormat::RGBA8, BGFX_TEXTURE_RT /*| BGFX_TEXTURE_SRGB*/ };
```

Comment:
If I uncomment `BGFX_TEXTURE_SRGB` then the rendering becomes wrong. I have not yet figured out why.
"